client/components/Treemap/TreemapSquare.tsx

```diff
@@ -9,7 +9,7 @@ type TreemapSquareProps = {
     'left' | 'top' | 'width' | 'height' | 'borderRadius'
   >
 
-function TreemapSquare({
+export function TreemapSquare({
```

Comment:
Why are we converting default exports to named once, that seems rather un-necessary.